src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/CodeGeneration/DefaultCodeRenderingContext.cs

```diff
@@ -73,7 +67,7 @@ internal class DefaultCodeRenderingContext : CodeRenderingContext
         if (newLineString != null)
         {
             // Set new line character to a specific string regardless of platform, for testing purposes.
-            codeWriter.NewLine = (string)newLineString;
+            CodeWriter.NewLine = (string)newLineString;
```

Comment:
Nit: would it make sense to figure out what the new line should be prior to creating code writer and pass in the correct value to the constructor? I.e. move line 49 after this if block?

Reply:
Possibly? I didn't really look much into this change as it indicated it was test-only.

Reply:
Read through the comment, and yeah, that would seem to make things cleaner. Went ahead and did it.